src/components/scenes/CreateWalletAccountSetupScene.tsx

```diff
@@ -1,203 +1,104 @@
-            <CryptoIcon marginRem={[1.5, 0, 0, 0]} pluginId={pluginId} sizeRem={4} tokenId={tokenId} />
-            <View style={[styles.createWalletPromptArea, { paddingTop: 24, paddingBottom: 8 }]}>
-              <Text style={styles.instructionalText}>{sprintf(lstrings.create_wallet_account_review_instructions, currencyCode)}</Text>
-            </View>
-            <View style={[styles.createWalletPromptArea, { paddingTop: 8, paddingBottom: 8 }]}>
-              <Text style={styles.handleRequirementsText}>{lstrings.create_wallet_account_requirements_eos}</Text>
-            </View>
+  useMount(() => logEvent('Activate_Wallet_Start'))
 
-            <View style={{ flexDirection: 'row', justifyContent: 'space-between' }}>
-              <OutlinedTextInput
-                autoFocus
-                autoCorrect={false}
-                onChangeText={this.handleChangeHandle}
-                label={lstrings.create_wallet_account_handle}
-                value={this.state.accountHandle}
-                returnKeyType="next"
-                onSubmitEditing={this.onSetup}
-                error={chooseHandleErrorMessage}
-              />
-              <View style={{ width: scale(25), height: scale(25) }}>
-                {isCheckingHandleAvailability ? (
-                  <ActivityIndicator color={THEME.COLORS.ACCENT_MINT} style={styles.feedbackIcon} />
-                ) : (
-                  <Image source={validityIcon} style={styles.feedbackIcon} />
-                )}
-              </View>
-            </View>
-            {showButton && this.renderButton()}
-            <View style={{ paddingBottom: 400 }} />
-          </View>
-        </ScrollView>
-      </SceneWrapper>
-    )
-  }
+  return (
+    <SceneWrapper scroll>
+      <View style={{ alignSelf: 'center' }}>
+        <CryptoIcon marginRem={1} pluginId={pluginId} sizeRem={4} tokenId={tokenId} />
+      </View>
+      {/* This is an abuse of ModalMessage,
+      but EdgeText breaks this text by setting numberOfLines.
+      Switch to MessageText if we ever define that: */}
+      <ModalMessage>{sprintf(lstrings.create_wallet_account_review_instructions, currencyCode)}</ModalMessage>
+      <ModalMessage>{lstrings.create_wallet_account_requirements_eos}</ModalMessage>
+      <OutlinedTextInput
+        autoCorrect={false}
+        autoCapitalize="none"
+        autoFocus
+        error={errorMessage}
+        label={lstrings.create_wallet_account_handle}
+        marginRem={[1, 1, 2]}
+        returnKeyType="next"
+        value={text}
+        onChangeText={handleChangeText}
+        onSubmitEditing={handleSubmit}
+      />
+      <MainButton
+        alignSelf="center"
+        disabled={spinning}
+        label={spinning ? undefined : lstrings.string_next_capitalized}
+        marginRem={0.5}
+        spinner={spinning}
+        type="primary"
+        onPress={handleSubmit}
+      />
+      <View style={{ paddingBottom: 400 }} />
```

Comment:
This hard-coded padding allows you to scroll the scene for no reason, especially apparent when the keyboard is collapsed.

The MainButton ought to be stuck to the bottom of the screen or top of the keyboard.

Reply:
However, this isn't possible unless we 
1. Disable scroll in the SceneWrapper or
2. Implement content size aware (dynamic scroll enable/disable) SceneWrapper or
3. Some other trick I never figured out

I'm fine punting this detail until we do 2